apps/base-docs/docs/connecting-to-the-blockchain/configuring-providers-sbs.md

```diff
@@ -0,0 +1,235 @@
+
+```typescript
+const { connectors } = getDefaultWallets({
+  appName: 'RainbowKit App',
+  projectId: 'YOUR_PROJECT_ID',
+  chains,
+});
+```
+
+:::caution
+
+Remember, anything you put on the frontend is public! That includes this id, even if you use environment variables to better manage this type of data. Next.js reminds you of the risk, by requiring you to prepend `NEXT_PUBLIC_` to any environment variables that can be read by the browser.
+
+Before you deploy, make sure you configure the rest of the items in the control panel to ensure only your site can use this id.
+
+:::
+
+### Public Provider
+
+By default, the setup script will configure your app to use the `publicProvider()`, and connect to a number of popular chains. To simply matters, remove all but `mainnnet`, `optimism`, and `base`.
+
+```typescript
+const { chains, publicClient, webSocketPublicClient } = configureChains(
+  [mainnet, optimism, base],
+  [publicProvider()],
+);
+```
+
+Open the terminal and start the app with:
+
+```bash
+yarn run dev
+```
+
+Click the `Connect Wallet` button, select your wallet from the modal, approve the connection, and you should see your network, token balance, and address or ENS name at the top of the screen. Select your wallet from the modal.
+
+![Rainbowkit modal](../../assets/images/connecting-to-the-blockchain/rainbowkit-modal.png)
+
+You've connected with the Public Provider!
+
+![Connected](../../assets/images/connecting-to-the-blockchain/connected.png)
+
+### QuickNode
+
+Notice that the [`configureChains`] function takes in an array of providers. It will attempt to use each one in the order provided, falling back to the next in the list in order if something goes wrong. You can use this for redundancy, or to smoothly handle scenarios where your app needs to connect with a chain that isn't supported by your preferred provider.
+
+Let's add [QuickNode] to the list. It isn't [included in the library] by default, so you'll need to add it using the generic [JSON RPC provider] import.
+
+```typescript
+import { jsonRpcProvider } from 'wagmi/providers/jsonRpc';
+```
+
+Unlike the linked example indicates, you do **not** need to add an import `chain` from `'wagmi'` (Their example appears out of date at the time of writing).
+
+You do need an RPC URL, so open up [QuickNode]'s site and sign up for an account if you need to. The free tier will be adequate for now, you may need to scroll down to see it. Once you're in, click `Endpoints` on the left side, then click `+ Create Endpoint`.
+
+On the next screen, you'll be asked to select a chain. Each endpoint only works for one. Select `Base`, click `Continue`.
+
+![Select Chain](../../assets/images/connecting-to-the-blockchain/quicknode-select-chain.png)
+
+For now, pick `Base Mainnet`, but you'll probably want to delete this endpoint and create a new one for Goerli or Sepolia when you start building. The free tier only allows you to have one at a time.
```

Comment:
Why not use testnet now for this lesson?

Reply:
No strong reason, other than it's already imported, and IMO neat to connect to the "real" thing.